templates/repo/settings/protected_branch.tmpl

```diff
@@ -5,38 +5,48 @@
 				{{.locale.Tr "repo.settings.branch_protection" (.Rule.RuleName|Escape) | Str2html}}
 			</h4>
 			<div class="ui attached segment branch-protection">
+				<h5 class="ui dividing header">{{.locale.Tr "repo.settings.protect_patterns"}}</h5>
 				<div class="field">
 					<label for="protected_file_patterns">{{.locale.Tr "repo.settings.protect_branch_name_pattern"}}</label>
 					<input name="rule_name" type="text" value="{{.Rule.RuleName}}">
 					<input name="rule_id" type="hidden" value="{{.Rule.ID}}">
 				</div>
-
-				<div class="ui divider"></div>
+				<div class="field">
+					<label for="protected_file_patterns">{{.locale.Tr "repo.settings.protect_protected_file_patterns"}}</label>
+					<input name="protected_file_patterns" id="protected_file_patterns" type="text" value="{{.Rule.ProtectedFilePatterns}}">
```

Comment:
No JS, no CSS, so these `for`/`id` could be removed now.

Reply:
removed in https://github.com/go-gitea/gitea/pull/24379/commits/db24ca8f8737d0cf2d7ace3b7f5d446a068f0bae